lms/djangoapps/courseware/tabs.py

```diff
@@ -304,7 +304,7 @@ def get_course_tab_list(request, course):
                 continue
             tab.name = _("Entrance Exam")
         if tab.type == 'static_tab' and tab.course_staff_only and \
-                not bool(user and CourseStaffRole(course.id).has_user(user)):
+                not bool(user and has_access(user, 'staff', course, course.id)):
```

Comment:
@lamagnifica and @sstack22 a side-effect of this change is that global staff will be able to see the "learner-hidden" page, regardless of whether or not they are staff on the course. This is consistent with the Instructor Dashboard, but I thought I would call it out (as I know the documentation was very careful to stay that only course staff would see the tab).

On the sandbox, admin@example.com (password edx) is a superuser.